website/src/components/IntroText/styles.module.css

```
@@ -0,0 +1,17 @@
+/* default styles (assuming light mode) */
+.introText {
+    color: #4B5563; /* default color approved by design (Madelyn) */
+    font-size: 1.15em;
+  }
+  
+  /* Explicitly set light mode styles */
+  html[data-theme='light'] .introText {
+    color: #4B5563; /* dark text in light mode approved by design (Madelyn)  */
+    font-size: 1.15em;
+  }
+  
+/* Adjusting for Dark Mode */
+html[data-theme='dark'] .introText {
+    color: #9CA3AF; /* white text in dark mode approved by design (Madelyn) */
+    font-size: 1.15em;
+  }
```

Since light-mode is the default, we can shorten up the css here a bit. The `data-theme='light'` block can be removed entirely, and light-mode will rely on the `.introText` styles. And since the `font-size` is already set within the default `.introText`, it can be removed from the `data-theme='dark'` block.

And if other styles override the `.introText` styles, you can add `html` before `.introText` to give it more specificity.

The updated css would look something like:
```suggestion
/* default styles (assuming light mode) */
html .introText {
  color: #4B5563;
  font-size: 1.15em;
}

/* Adjusting for Dark Mode */
html[data-theme='dark'] .introText {
  color: #9CA3AF;
}
```